views/update/remove-confirm-statement.html

```diff
@@ -0,0 +1,49 @@
+{% extends "update-layout.html" %}
+
+{% set title = "Confirm the removal statement - Apply to remove and overseas entity" %}
```

Comment:
Ah yes, so text looks wrong here too. Does it come from the prototype? Might need to clarify with analyst and UI team.

Reply:
As above